include/glow/Base/Tensor.h

```diff
@@ -315,6 +315,11 @@ class Tensor final {
       return isEqualImpl<int32_t>(other, allowedError);
     case ElemKind::Int64ITy:
       return isEqualImpl<int64_t>(other, allowedError);
+      // Note: We can use isEqualImpl() here because the scales/offsets will be
+      // compared as if they were data, so we will return false if any rowwise
+      // scale/offset do not match.
+    case ElemKind::Int8FusedQTy:
```

Comment:
having this, `isEqual` became a little bit misleading since it's possible that Int8Fused and Int8QT tensors will be equal, but just written in different representation.
I'm ok if we address that later, just put some more details on the issue about comparing two quantized tensors.

Reply:
I added added an assertion and a comment for this.